### DISCUSSION TITLE
How to re open automatic password saving?

### DISCUSSION THREAD

**Discussion by 9854626** [MicrosoftEdge/WebView2Feedback#3998] [Q&A]
Hello, I want to know that after opening automatic password saving, I clicked Never when the password saving prompt was displayed on the page. How can I turn on the switch again?
![26A8C8B7-E750-4ae5-9C40-678159F0F39D](https://user-images.githubusercontent.com/49057770/202951151-2c27a0bd-2fdf-4579-ba25-35d33f29a346.png)

**Comment by victorthoang** [comment]
Hello @TempleToth,

Can you confirm that your question is related to WebView2, and not the Edge browser?

Thank you!

**Comment by 9854626** [comment]
Yes, I use  WebView 2 `CoreWebView2Settings.IsPasswordAutosaveEnabled` to open this option. But after clicking Never, I can't open this setting again.

**Comment by victorhuangwq** [comment]
May I ask if you are still facing this issue?
